wordpress-mobile/WordPress-Android
# Feature/quick actions v1

This PR adds the first iteration of static Quick Actions. 

PR looks a bit big, but the majority of changes is from moving things around.

To test:
- Make sure quick action buttons work ok.
- Quick Start, pointers point to the correct rows and blavatar.
- Tracking events are getting hit.
- Nothing looks weird.
- Clicking on Site Name or URL opens site preview.
- Users without access to pages should not see Pages quick action.

![63117921-9b48b380-bf51-11e9-9eb4-14dcb514ade1](https://user-images.githubusercontent.com/728822/63399349-190f2380-c385-11e9-80dc-759bf5eb6505.png)

Update release notes:
